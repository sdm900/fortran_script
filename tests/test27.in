do i=1,10
  a(i) = -1+i
  if(a(i)==6) print "got it", i
enddo

print a(7)
    
