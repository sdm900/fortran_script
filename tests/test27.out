got it     7.00000 
     6.00000 
